components/ui/multi-select.tsx

```
@@ -0,0 +1,394 @@
+} from "react";
+
+interface MultiSelectorProps
+    extends React.ComponentPropsWithoutRef<typeof CommandPrimitive> {
+    values: string[];
+    onValuesChange: (value: string[]) => void;
+    loop?: boolean;
+}
+
+interface MultiSelectContextProps {
+    value: string[];
+    onValueChange: (value: any) => void;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)